### DISCUSSION TITLE
Preferred colours for scenery or staff

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#20097] [Feature requests]
Whom would like to define which colours should be default for staff or scenery objects?

**Comment by MASHzj** [comment]
It's not very clear to me what you mean

- For staff: default colours can differ per scenario, and the default is red for handymen and blue for mechanics. This is fine in my opinion.

- For scenery, do you mean to have different default colours for different scenery objects? Or are you just talking about the default colours for scenery in general? In the latter case, I saw discussion on Discord where the conclusion was to leave the default colours as they are, but with a possibility to save the last used colours.

**Comment by 733737** [comment]
Even if default staff colors is per scenario, you could still override this optionally with your own default colors. Scenery is always bordeaux red and yellow, I am fine with saving the last used if people aren't interested in random colors or something other than yellow and bordeaux red.
